### ISSUE TITLE
Neural Network interactive tweaks.

### ISSUE THREAD

**Issue by 3b1b** [3b1b/3Blue1Brown.com#169]
1) On mobile, the interaction is overridden by scrolling, so it's not possible to draw vertical lines.

---

2) So as to give a better "this can actually work" feeling, we should figure out what the right way to normalize drawings is to better match what the network was trained on.

As one anecdotal example, I've noticed that 7's (or at least my 7's) are often poorly recognized, but if I write it lower in the panel that changes.

7 written high
![image](https://user-images.githubusercontent.com/11601040/124224399-6c427900-daba-11eb-8b75-cbf858c1333b.png)

7 written lower
![image](https://user-images.githubusercontent.com/11601040/124224440-7feddf80-daba-11eb-8fa7-d22c62ba1b50.png)

One potential strategy could be to go through the training data, and see the average buffer from the left, right, top and bottom. That buffer, say from the left, might be measured as the number of totally blank columns on the left, or perhaps the number of columns until one which has at least one pixel with >0.5 brightness.

Then we could have a "pre-process" button that shifts and scales the drawn digit appropriately to match those statistics. Shifting is easy, scaling might be a little tricky. One method could be to store the list of 2d points for where the cursor was during clicking. Those points can easily be scaled and shifted, and the actual painting of the pixels can be redrawn from there.

---

3) I think we should enable the see-the-network-change-as-you-draw mode on all version of it, say as a subtle button which can be toggled on or off.

---

4) There should be a full-screen button.

---

5) (Not a suggestion, just a comment) I keep having way too much fun playing with this.

**Comment by PullJosh** [comment]
I have a pre-process button working and it is shockingly effective, which is exciting.

There is definitely room to write more about the need for pre-processing in the network analysis mini-lesson.
